### ISSUE TITLE
Web Socket not pulling Questions & Answers on 3/9/18 @ 9PM EST Show

### ISSUE THREAD

**Issue by MxQuid** [DanielSmith1239/HQTriviaMaster#67]
I used HQ Trivia Master tonight for the 9pm EST show, but it wasn't able to pull the questions or answers. This was my first time trying the Web Socket instead of OCR. As far as I can tell from MITMProxy and Xcode Debugger & Console, there seems to be no issues that I can tell. I have successfully used OCR mode before, so I know API and CSE keys are input properly. I have retrieved my user ID and token via MITMProxy, and I think I input them properly into HQTM.

I have two questions:
1) My user ID is supposed to be 8 digits, correct? And how is the token supposed to be formatted? Do I leave Bearer in? Do I ensure the token has no spaces or returns in it?

2) Is it possible that HQ Trivia changed some things, throwing off the Web Socket?

**Comment by DanielSmith1239** [comment]
1. not sure; either way; just copy and paste it
2. nope, works fine for me

**Comment by MxQuid** [comment]
For those of you who may struggle to configure the Web Socket, here are some things I had realized:
1) As Daniel stated, a simple copy and paste of the token, with or without Bearer, should do the trick. I personally removed the returns/enters and spaces between and at the end of the token since copying & pasting from MITMProxy was kind of wonky for me. I did, however, leave the spaces between bearer and the start of the token string.

2) Ensure that you removed proxy settings from your iOS/Android device after retrieving your token, as I overlooked this step in the instructions. Remember, you don't need to have MITMProxy open once you retrieve your User ID and token.

And not 100% related to the web socket specifically, remember to add multiple API keys to ensure that you're getting legitimate results from Google. I pretty quickly started getting false-positives before I added an additional API key while doing some testing with YouTube videos and OCR.
